CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## unreleased
 
+## v4.5.1 - 2023.03.03
```

Comment:
Please omit the header -- that's going to be added at the time we do the release for sure.

Reply:
I've removed the version header.